### ISSUE TITLE
Does it support mixed image/video galleries?

### ISSUE THREAD

**Issue by ScotsScripts** [vvvmax/unitegallery#195]
Just curious, trying to get images and videos in same gallery without luck. Wondered if the gallery supports this? Thanks.

**Comment by vvvmax** [comment]
Yes it supports

**Comment by ScotsScripts** [comment]
Nice! Is there an example somewhere? Thanks for putting together what I think is the best image gallery plugin out there.

**Comment by vvvmax** [comment]
Yes there are examples on my site. It called mixed gallery

**Comment by ScotsScripts** [comment]
Thanks again. 

I got it working, my fault, typo - : is not the same as ;

Sheesh.

**Comment by ScotsScripts** [comment]
Ok, the vids work in the slideshow (I'm testing on grid style) but I can't get the thumbnails to show up. Do vid thumbs only work in tile mode?

**Comment by vvvmax** [comment]
It should work :) i cant do tech support here sorry

**Comment by ScotsScripts** [comment]
Yeah, I understand. I'm just trying to determine if this is a bug so I can stop spending time on it.

data-thumb works perfect for videos in "tiles" mode, but not in any other mode.  I tried with 1.7.40 from cloudflare cdn and also downloaded 1.7.45 and tried that.

Is there a setting to force data-thumb work in other modes?

**Comment by vvvmax** [comment]
Should work for all modes. Its the same functiin for every gallery mode.

**Comment by ScotsScripts** [comment]
Ok, ok, I got it- to add videos to a mixed gallery you gotta use the IMG tag method on the docs page. For videos in a video-only gallery then you use the DIV tag setup in the video syntax option in the video tab. 

It would be nice if the thumbnail and video image were created dynamically the way they are for video galleries though.

**Comment by vvvmax** [comment]
It will slow it down. Sorry.

**Comment by ScotsScripts** [comment]
Off topic: do you have plan to put 1.7.45 on cloudflare cdn?

**Comment by vvvmax** [comment]
Its automatic

**Comment by ScotsScripts** [comment]
Hmm, it's not there. Last version on cloudflare is 1.7.40

**Comment by vvvmax** [comment]
Dont know man. Its taking auto from github

**Comment by ScotsScripts** [comment]
I'm guessing it's because the releases page only goes to 1.7.40

https://github.com/vvvmax/unitegallery/releases

**Comment by vvvmax** [comment]
yea, you right, not priority for me thought :)

**Comment by ScotsScripts** [comment]
I get it, I'll use 1.7.40 until the cdn is updated. Thanks for all your help!
